app/mender_test.go

```diff
@@ -756,7 +778,7 @@ echo foo=bar`),
 	// 3. pretend client is no longer authorized
 	srv.Auth.Token = []byte("footoken")
 	err = mender.InventoryRefresh()
-	assert.NotNil(t, err)
```

Comment:
Question:
Was this wrong previously?

Reply:
No, is just that checking `err` is not nil covers the same as checking that `err` is an `Error` (while IMO the later is more meaningful)

Reply:
The assertion is identical, it just makes it more obvious if it fails due to an error.